src/components/Project/AragonClient.js

```diff
@@ -493,14 +510,16 @@ class AragonClient extends React.Component {
                 <Module>
                   <div className="badge-container">
                     <StyledBadge className="grey">
-                      Developer resources
+                      Security
                     </StyledBadge>
                     <AragonOneBadge>Aragon One</AragonOneBadge>
                   </div>
                   <h3>aragonOS: Global emergency failsafe</h3>
                   <h4>
-                    Provide an opt-in failsafe for organizations and individual
-                    apps to safely freeze their state during times of emergency.
+                    Provide an opt-in failsafe for the smart contracts of organizations
+                    to safely shut down the organization should a critical smart contract bug be
```

Comment:
>  to safely shut down the organization

isn't the original "freeze" more accurate, or at least not as drastic sounding? "Shut down" sounds a bit extreme, like if I "shut down" my company that usually means it's indefinitely closed.